src/extension_commands.ml

```diff
@@ -56,6 +56,28 @@ let _select_sandbox =
   command Extension_consts.Commands.select_sandbox handler
 ;;
 
+let _install_ocaml_lsp_server =
+  let handler (instance : Extension_instance.t) ~args:_ =
+    let open Promise.Syntax in
+    let (_ : unit Promise.t) =
```

Comment:
I think that type anotation is not useful?

Reply:
@smorimoto 
Ok, since it is wrapped into a comprehension (via `let*`), inference works pretty well, so in this review, FMPOV, it is noise, but my goal is not to change the full code-base :)